reactDashboard/src/pages/Regional.jsx

```diff
@@ -87,55 +88,64 @@ const Regional = ({
   }, [regionCode, location]);
 
   return (
-    <Container fluid className="regional-page">
-      <Row className="d-none d-lg-block">
-        <Col>
-          <hr className="full-width-hr" />
-        </Col>
-      </Row>
-      <Row>
-        <Col xs={12} lg={4}>
-          <RegionGeoMap
-            councilAreaDataset={councilAreaDataset}
-            healthBoardDataset={healthBoardDataset}
-            regionCode={regionCode}
-            setRegionCode={setRegionCode}
-          />
-        </Col>
-        <Col xs={12} lg={8}>
-          <hr className="d-flex d-md-none full-width-hr" />
-          <strong>Select region (or select on map):</strong>
+    <>
```

Comment:
I am not sure if <> was the right way to go as Overview.jsx instead wraps the render in a <Container>. If this is a problem, let me know.

Reply:
Meh - its fine - it matches overview.jsx at least